templates/eventwizard.html

```diff
@@ -0,0 +1,156 @@
+{% extends "base.html" %}
+
+{% block title %}The Blue Alliance - Event Wizard{% endblock %}
+
+{% block meta_description %}Interface to update matches using The Blue Alliance's write API.{% endblock %}
+
+{% block content %}
+<div class="container">
+  <div class="row">
+    <div class="col-xs-12">
+      <h1 class="endheader">TBA Event Wizard</h1>
+      <p>A tool to help you archive all the data associated with an event</p>
```

Comment:
Change to "A tool to input event data. Contact (link) if you're running an offseason event and would like to use this tool."